fontData.js

```diff
@@ -12,6 +12,76 @@ const buildAxes = font => {
   }));
 };
 
+const guessFontStyle = fontName => {
+  const styles = [
+    "italic",
+    "thin",
+    "hairline",
+    "extra-light",
+    "extra light",
+    "extralight",
+    "ultra-light",
+    "ultra light",
+    "ultralight",
+    "light",
+    "normal",
+    "regular",
+    "medium",
+    "semi-bold",
+    "semi bold",
+    "semibold",
+    "demi-bold",
+    "demi bold",
+    "demibold",
+    "bold",
+    "extra-bold",
+    "extra bold",
+    "extrabold",
+    "ultra-bold",
+    "ultra bold",
+    "ultrabold",
+    "black",
+    "heavy",
+    "extra-black",
+    "extra black",
+    "extrablack",
+    "ultra-black",
+    "ultra black",
+    "ultrablack",
+    "ultra-condensed",
+    "ultra condensed",
+    "ultracondensed",
+    "extra-condensed",
+    "extra condensed",
+    "extracondensed",
+    "condensed",
+    "semi-condensed",
+    "semi condensed",
+    "semicondensed",
+    "normal",
+    "semi-expanded",
+    "semi expanded",
+    "semiexpanded",
+    "expanded",
+    "extra-expanded",
+    "extra expanded",
+    "extraexpanded",
+    "ultra-expanded",
+    "ultra expanded",
+    "ultraexpanded"
+  ];
+
+  const matches = styles.filter(style =>
+    fontName.toLowerCase().includes(style) ? style : false
+  );
+
+  if (matches.length) {
+    return matches.join(" ");
```

Comment:
What do you think of returning an array here? That way it's a little more flexible for other use cases.

Reply:
Yeah, perhaps. It would be nicer to split a group of fonts into roman (...light, regular, bold) and italic (...light, regular, bold) so you have more control for split/sort/group-ing. But then again, don't want to overcomplicate this. Let me think about it!

Reply:
On second thought, for now I think a string will suffice, for [this](https://github.com/kabisa/specimen-skeleton-support/pull/14#discussion_r497329341) reason!